_posts/2014-11-13-cockpit-plugin-tutorial.md

```diff
@@ -63,7 +62,7 @@ pinger: /home/.../.local/share/cockpit/pinger
 ...
 ```
```

Comment:
```suggestion
```

You should see `pinger` listed among all the active modules, like the above example.
```